.github/workflows/cypress.yml

```
@@ -0,0 +1,79 @@
+name: E2E test
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    uses: 10up/embed-block-figma/.github/workflows/build-release-zip.yml@develop
+  cypress:
+    needs: build
+    name: ${{ matrix.core.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        core:
+          - {name: 'WP latest', version: 'latest'}
+          - {name: 'WP minimum', version: 'WordPress/WordPress#5.7'}
```

Our minimum WP version for this plugin is 6.4
```suggestion
          - {name: 'WP minimum', version: 'WordPress/WordPress#6.4'}
```